src/main/java/stsjorbsmod/powers/CatharsisPower.java

```diff
@@ -0,0 +1,59 @@
+package stsjorbsmod.powers;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageRandomEnemyAction;
+import com.megacrit.cardcrawl.actions.common.LoseHPAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import com.megacrit.cardcrawl.relics.BlueCandle;
+
+public class CatharsisPower extends CustomJorbsModPower {
+    public static final StaticPowerInfo STATIC = StaticPowerInfo.Load(CatharsisPower.class);
+    public static final String POWER_ID = STATIC.ID;
+
+    private static final int HP_LOSS = 1;
+
+    public CatharsisPower(final AbstractCreature owner, final int damagePerCurse) {
+        super(STATIC);
+
+        this.owner = owner;
+        this.amount = damagePerCurse;
+        this.amount2 = 0;
```

Comment:
I'm not sure tracking as amount2 is ideal here; I think the wording of the power suggests that it should be counting cases where you played curses with blue candle/pride before playing the power. I think maybe querying `AbstractDungeon.actionManager.cardsPlayedThisCombat` would be a better way to go